tdms/include/globals.h

```diff
@@ -50,19 +50,21 @@ enum RCSType   { parallel , perpendicular };
 //			Mathematical Constants
 // **************************************
 
-const double dcpi  = 3.14159265358979323846;
-
+namespace tdms_math_constants
+{
+const double dcpi = 3.14159265358979323846;                                  // Pi
+const std::complex<double> IMAGINARY_UNIT = std::complex<double>(0.0, 1.0);  // Imaginary unit
+}

```

Comment:
Relavent Google style recommendations:

* [Namespace naming](https://google.github.io/styleguide/cppguide.html#Namespace_Names) (✅)
* [Namespace formatting](https://google.github.io/styleguide/cppguide.html#Namespace_Formatting)
* [const → constexpr](https://google.github.io/styleguide/cppguide.html#Use_of_constexpr)
* [Const naming](https://google.github.io/styleguide/cppguide.html#Constant_Names)

A bit annoying: as a global `IMAGINARY_UNIT` is what I'm used to. But as a const(expr?) in a namespace, it seems GSG prefers `kImaginaryUnit`. I'm not super opinionated on this point, and `readability counts` 🐍 .

I found [this scicomp stackexchange post](https://scicomp.stackexchange.com/a/31003) (there is a scicomp stackexchange!). Certainly I'm more used to what is suggested there:

```c++

namespace tdms_math_constants {

constexpr double PI = 3.14159265358979323846;                                    // Pi
constexpr std::complex<double> IMAGINARY_UNIT = std::complex<double>(0.0, 1.0);  // Imaginary unit

}

//--

namespace tdms_phys_constants {

constexpr double E0 = 8.85400e-12;                             // free space electric permitivity
constexpr double M0 = 4.0 * tdms_math_constants::PI * 1.0e-7;  // free space magnetic permeability
constexpr double LIGHT_V = 1.0 / sqrt(M0 * E0);                // free space light velocity
constexpr double Z0 = 376.734;                                 // free space inpedance

}
```

whereas GSG would have us do:

```c++

namespace tdms_math_constants {

constexpr double kPi = 3.14159265358979323846;                                   // Pi
constexpr std::complex<double> kImaginaryUnit = std::complex<double>(0.0, 1.0);  // Imaginary unit

}

//--

namespace tdms_phys_constants {

constexpr double kE0 = 8.85400e-12;                              // free space electric permitivity
constexpr double kM0 = 4.0 * tdms_math_constants::kPi * 1.0e-7;  // free space magnetic permeability
constexpr double kLightV = 1.0 / sqrt(M0 * E0);                  // free space light velocity
constexpr double kZ0 = 376.734;                                  // free space inpedance

}
```

At this point, I'd simply pick your favourite and be consistent. But I wouldn't mix variable naming styles for the same thing (as you have right now). Let's get it right once and forall now that we're making a noisy "touches many files" -type PR.

Reply:
The mixture is present in the original code - I'll convert everything to SciComp standards because that's what I'm used to seeing too. This PR is going to look a lot bigger after these changes just because of all the name changes...